Rakefile

```diff
@@ -112,6 +112,14 @@ task 'gcovr' => 'check-cover' do
   end
 end
 
+task :afl_flags => :bpnode_size_4 do
+    ENV['REALM_AFL'] = '1'
+    ENV['REALM_ENABLE_ENCRYPTION'] = '1'
+end
+
+desc 'Build and instrument the code for fuzz testing with AFL.'
```

Comment:
I guess it doesn't actually run the fuzz test?

Also I'm wondering if AFL-instrumented code shouldn't be release mode. AFL-instrumented code should be as fast as possible to allow for the highest possible iteration speed, and then the generated repro cases can be used with a debug build to investigate whatever problems exist (at least this is how I've been using it).

Not a dealbreaker at all, just some thoughts, take it or leave it. :-)